releasenotes/notes/remove-execute_function-max_credits-8c822b8b4b3d84ba.yaml

```
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    The function `qiskit.execute_function.execute` and `qiskit.compiler.assembler.assemble`
+    deprecated the argument `max_credits` and now is being removed.
```

```suggestion
    The ``max_credits`` argument to :func:`~.execute_function.execute`, :func:`~.compiler.assemble` and all
    of the ``Qobj`` configurations (e.g. :class:`.QasmQobjConfig` and
    :class:`.PulseQobjConfig`) has been removed.
    The credit system has not been in use on IBM Quantum backends for
    nearly three years, and the option has no effect.  No alternative is necessary.
```

(matches more what we put in the original note)